scripts/demo.py

```diff
@@ -38,6 +38,7 @@ def demo(model):
 if __name__ == "__main__":
     print("Torch version:", torch.__version__)
     model = UniDepthV1.from_pretrained("lpiccinelli/unidepth-v1-vitl14")
+    # model = UniDepthV2.from_pretrained("lpiccinelli/unidepth-v2-vitl14")
```

Comment:
Which one should we use for the demo by default? We could always use the latest version, right?

Reply:
This is just to make people aware of it